pkg/apiserver/handlers.go

```diff
@@ -139,6 +139,13 @@ func tooManyRequests(req *http.Request, w http.ResponseWriter) {
 func RecoverPanics(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		defer runtime.HandleCrash(func(err interface{}) {
+			if e, ok := err.(error); ok {
```

Comment:
It is up to you I guess. I checked again, now there is no other panic cases than the one we added in this commit. Just feel that handling it in recoverPanics is a little bit better.